python/cugraph/cugraph/link_prediction/jaccard.py

```diff
@@ -106,8 +105,8 @@ def jaccard(input_graph, vertex_pair=None):
     >>> G.from_cudf_edgelist(gdf, source='0', destination='1')
     >>> df = cugraph.jaccard(G)
     """
-    if type(input_graph) is not Graph:
-        raise Exception("input graph must be undirected")
+    if type(input_graph) is DiGraph:
+        raise TypeError("input graph must a Graph")
```

Comment:
```suggestion
    if type(input_graph) is not Graph:
        raise TypeError("input graph must a Graph")
```
This is better since there are other graph types besides just Graph and DiGraph.  You'll also have to change the import of `DiGraph` above.

Reply:
sorry if this is confusing.  I noticed the last change was to use `DiGraph`, but my prior review meant to simply call `is_directed()` to cover more cases where a directed graph that wasn't a Graph type (eg. something that inherits from `Graph`) could be supported.

This could be harder though since you may get an instance that doesn't have a `is_directed` method, in which case you could do:
`if not(hasattr(input_graph, "is_directed")) or (input_graph.is_directed() is False):  raise TypeError...`

I'm fine with either `is not Graph` or the above for now.